docs/source/ui/pages/reports.md

```diff
@@ -0,0 +1,175 @@
+
+The results of this 
+Example:
+
+```python
+```aim
+metrics = repo.fetch_metrics('metric.name == "loss"')
+linechart = LineChart(metrics)
+```\
+```
+
+
+### Visualization objects and methods
+
+- `LineChart`
+
+Signature:
+`LineChart(data, x, y)`
+
+Parameters:
+
+| name | type | default | default | 
+|-----|------------|---------|-------|
+| `data` | `List[dict]` | required| the data to be plotted |
+| `x`| `str` | `steps` |  the path to the property for the x-axis value (for fetched metrics available options are `epochs`, `timestamps`)|
+| `y`| `str` | `values`| the path to the property for the y-axis value |
+
+Example:
+
+```aim
+metrics = repo.fetch_metrics()
+linechart = LineChart(metrics, x='timestamps')
+```
+
+- `ImagesList`
+
+Signature:
+`ImagesList(data)`
+
+<img alt="Images List" style="border-radius: 8px; border: 1px solid #E8F1FC" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/reports/images-list.png">
+
+Parameters:
+
+| name | type | default | default | 
+|-----|------------|---------|-------|
+| `data` | `List[dict]` | required| the data to be plotted |
+
+Example:
+
+```
+images = repo.fetch_images()
+ImagesList(images)
+```
+
+
+- `AudiosList`
+
+Signature:
+`AudiosList(data)`
+
+Parameters:
+
+| name | type | default | default | 
+|-----|------------|---------|-------|
+| `data` | `List[dict]` | required | the data to be plotted |
+
+Example:
+```
+audios = repo.fetch_audios()
+AudiosList(audios)
+```
+- `FiguresList`
+
+Signature:
+`FiguresList(data)`
+
+Parameters:
+
+| name | type | default | default | 
+|-----|------------|---------|-------|
+| `data` | `List[dict]` | required| the data to be plotted |
+
+Example:
+```
+figures = repo.fetch_figures()
+FiguresList(figures)
+```
+- `TextsList`
+
+Signature:
+`TextsList(data)`
+
+Parameters:
+
+| name | type | default | default | 
+|-----|------------|---------|-------|
+| `data` | `List[dict]` | required| the data to be plotted |
+
+
+ Example:
+```
+texts = repo.fetch_texts()
+TextsList(texts)
+```
+
+
+All the aforementioned objects have `group` method available to them:
+
+```python
+def group(prop, values=[]):
+    ...
+```
+`prop`- name of the property to be grouped by. Available options are: `color`, `stroke_style`, `row`, `column`
+
+`value`- list of values of sequence fields to be grouped by. Available fields are all those fields that are also available in grouping options of the explorer pages.
+
+The `group` method can be applied multiple times sequentially.
+
+Example:
+```
+metrics = repo.fetch_metrics()
+linechart = LineChart(metrics)
+linechart.group('color', ['run.hash'])
+linechart.group('row', ['metric_name'])
+```
+
+<img alt="Grouping Example" style="border-radius: 8px; border: 1px solid #E8F1FC" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/reports/grouping-example.png">
+
+### Search Report 
+You can easily find the report in a big pile with the search ba.
```

Comment:
Typo: bar

Reply:
In general instead I would write:
The search bar is available so you can easily find the report you are looking for.